lib/typesafe_enum/module_info.rb

```diff
@@ -5,8 +5,5 @@ module TypesafeEnum
   NAME = 'typesafe_enum'
 
   # The version of this gem
-  VERSION = '0.3.0'
-
-  # The copyright notice for this gem
-  COPYRIGHT = 'Copyright (c) 2020 The Regents of the University of California'
```

Comment:
It's an MIT license.

Reply:
Rad. Two things in this block.

- let's just bump minor version (0.4.0) since this just adds features and doesn't necessarily flip a "stability bit" (https://www.youtube.com/watch?v=p_Rpkaoo8M8)
- let's leave the copyright thing in there, for two reasons:
  * MIT License terms state that we have to preserve their copyright header anyway. If we change this at all, we should only change it to add `& The Wanderlust Group` or some such to the end (or a new line, or whatever)
  * It keeps the diff less disruptive if upstream is responsive to an attempt to PR it upwards rather than maintain our own diff

Reply:
Sweet! Sounds great. Good points